core/state_transition.go

```diff
@@ -128,6 +128,15 @@ func toWordSize(size uint64) uint64 {
 	return (size + 31) / 32
 }
 
+type RunMode uint8
+
+const (
+	CommitMode RunMode = iota
```

Comment:
Please assign `CommitMode` to `RunMode` in function `TransactionToMessage`. Not a bug. It is strange that `CommitMode` is never used.

Reply:
fixed, set default `RunMode` as `CommitMode`